assignments/objects.js

```diff
@@ -18,47 +19,112 @@ const example = {
 }
 
 // Write your intern objects here:
+const newInterns = {
+ 0:{
+  "id": 1,
+  "email": "mmelloy0@psu.edu",
```

Comment:
You could achieve a similar result by declaring an array of objects (so that you won't have to assign numerical indices).